auto_nag/scripts/copy_duplicate_info.py

```diff
@@ -117,6 +117,10 @@ def compare(self, bugs, dups):
 
             dup = dups[dupid]
             bs = utils.get_signatures(info["signature"])
+            if dup["signature"] is None:
+                # some bugs don't have a cf_crash_signature field... it's weird
```

Comment:
Did you ask dkl about this?

Reply:
yes in #bmo on slack.

Reply:
What was the problem? Is it a bug or a feature? :P

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1742458